res/parking/parking_manager.c

```diff
@@ -557,7 +558,13 @@ static int manager_park(struct mansession *s, const struct message *m)
 		ast_bridge_set_transfer_variables(chan, timeout_channel, 0);
 		ast_channel_unlock(chan);
 	}
-
+	
+	if (!ast_strlen_zero(parkingpos)) {
+		ast_channel_lock(chan);
```

Comment:
pbx_builtin_setvar_helper locks the channel, you don't also need to do that here.